DataFormats/Portable/test/alpaka/test_catch2_heterogeneousDeepCopy.dev.cc

```diff
@@ -0,0 +1,301 @@
+#include <Eigen/Core>
+#include <Eigen/Dense>
+
+#include <alpaka/alpaka.hpp>
+
+#define CATCH_CONFIG_MAIN
+#include <catch.hpp>
+
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+#include "DataFormats/Portable/interface/PortableCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+
+using namespace ALPAKA_ACCELERATOR_NAMESPACE;
+
+#define TIME 0.01
```

Comment:
Does this really need to be a preprocessor symbol, or could it be
```suggestion
constexpr float interval = 0.01f;
```
?